common/views/components/FeaturedCard/FeaturedCard.js

```diff
@@ -0,0 +1,223 @@
+// @flow
+import { type Node } from 'react';
+import NextLink from 'next/link';
+import { type UiImageProps, UiImage } from '../../components/Images/Images';
+import type { UiExhibition } from '../../../../common/model/exhibitions';
+import type { Article } from '../../../../common/model/articles';
+import { type Label } from '../../../../common/model/labels';
+import { type Link } from '../../../../common/model/link';
+import PartNumberIndicator from '../../components/PartNumberIndicator/PartNumberIndicator';
+import { grid, classNames, font } from '../../../utils/classnames';
+import Space from '../styled/Space';
+import styled from 'styled-components';
+import LabelsList from '../LabelsList/LabelsList';
+import StatusIndicator from '../../components/StatusIndicator/StatusIndicator';
+import { formatDate } from '../../../../common/utils/format-date';
+import {
+  getPositionInSeries,
+  getArticleColor,
+} from '../../../../common/model/articles';
+import { trackEvent } from '../../../utils/ga';
+
+type Props = {|
+  id: string,
+  image: ?UiImageProps,
+  labels: ?(Label[]),
```

Comment:
Arrays should never really be nullable, could we just make this a `Label[]`.

The reason being is you land up having to do weird stuff like:
```
labels && labels.length > 0 && labels.map()
```